esp/files/css/espdefault.css

```diff
@@ -66,12 +66,12 @@ a:active.espnavlink {
 .espnavcollapse 
 {
     width:18px;top:0;right:1;height:15px;position:absolute;zoom:1;
-    background:transparent url(/esp/files/yui/build/layout/assets/skins/sam/layout_sprite.png) no-repeat -20px -120px;border:1px solid #808080;
+    background:transparent url(../yui/build/layout/assets/skins/sam/layout_sprite.png) no-repeat -20px -120px;border:1px solid #808080;
```

Comment:
Why not use files_/yui/... as other places?

Reply:
I think in this case it's because this file is found deeper down the path, in the css folder.  The .. means back up one level.  The paths that use files_ are being called as part of a service form or similar.

Reply:
To better explain, I think from here if you said "file_/yui/build/layout/assets/skins/sam/layout_sprite.png"
The resulting path would be:
/esp/files/css/files_/yui/build/layout/assets/skins/sam/layout_sprite.png

Reply:
Based on that logic, the new path inside the esp/xslt/appframe.xsl should be: /esp/files/xslt/files_/yui/build/fonts/fonts-min.css. Is that what we want (or files_ = ../files)?

Reply:
Correct, using files_/...etc... results in a this as the submitted URL:
/hpcc/esp/files_/css/files_/yui/build/layout/assets/skins/sam/layout_sprite.png